app/helpers/budget_bar_helper.rb

```diff
@@ -5,15 +5,30 @@ def progress_bar_color(type)
     "progress-bar-#{color}"
   end
 
-  def progress_bar_width(value)
-    "width: #{to_pct(value)}%"
+  def progress_bar_width(value, total = 1_000)
```

Comment:
Is the default `total` here meant to match `default_cash_ceiling` in the dashboard_controller (and will thus later be linked), or is this coincidence? If the former, I'd tag a comment here so we don't miss it later.

Reply:
This is a fabulous catch, they're intended to match. I'll change these vars to match.